spec/config/initializers/ab_tests_spec.rb

```diff
@@ -7,7 +7,7 @@
         {
           ACUANT_SDK: an_instance_of(AbTest),
           DOC_AUTH_VENDOR: an_instance_of(AbTest),
-
+          SOCURE: an_instance_of(AbTest),
```

Comment:
this seems like a silly test to me because every addition/removal of an AB test (which should be lightweight) requires adjusting the specs.

I think it would be better served by something like:

```suggestion
      expect(AbTests.all.values).to all(be_kind_of(AbTest))
```

Reply:
Agree - updated in 11dcfb1